mix/CodeModel.h

```diff
@@ -138,7 +138,7 @@ class GasMapWrapper: public QObject
 
 public:
 	GasMapWrapper(QObject* _parent = nullptr): QObject(_parent){}
-	void push(QString _source, int _start, int _end, QString _value, bool _isInfinite);
+	void push(QString _source, int _start, int _end, QString _value, bool _isInfinite, QString _payload);
```

Comment:
Better not use magic strings in C++ code. Store an enum and convert to string in property getter